config-connector/solutions/projects/kpt/owned-project/Kptfile

```diff
@@ -0,0 +1,36 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: owned-project
+packageMetadata:
+  shortDescription: Deploy a project under a folder with given IAM owner
```

Comment:
Maybe it's clearer to say "Create a project under a folder and grant permissions for the project"?